website/utils.py

```
@@ -1129,3 +1129,222 @@ def get_default_bacon_score(model_name, is_security=False):
+            return None
+
+    except Exception as e:
+        logging.error(
+            f"Error processing bug screenshot {getattr(image_file, 'name', 'unknown')}: {str(e)}", exc_info=True
+        )
+        return None
+    finally:
+        # Reset file pointer for potential reuse
+        try:
+            image_file.seek(0)
+        except:
```

Ruff (0.14.7)

Do not use bare `except`

(E722)